src/installer.ts

```diff
@@ -136,7 +136,13 @@ async function computeVersion(
 
 function getFileName() {
   const platform: string = osPlat === "win32" ? "windows" : osPlat;
-  const arch: string = osArch === "x64" ? "amd64" : "386";
+  const arches = new Map<string, string>([
+    ["arm", "arm"],
+    ["arm64", "arm64"],
+    ["x64", "amd64"],
+    ["ia32", "386"],
+  ]);
+  const arch: string = arches.get(osArch) ?? osArch;
```

Comment:
Nit: I think a simple object stays more readable and we don't need the map api/performance
Granted that typescript doesn't scream

```suggestion
  const arches = {
    arm: "arm",
    arm64: "arm64",
    x64: "amd64",
    ia32: "386",
}
  const arch: string = arches[osArch] ?? osArch;
```